packages/grafana-data/src/themes/createSpacing.ts

```diff
@@ -67,5 +77,26 @@ export function createSpacing(options: ThemeSpacingOptions = {}): ThemeSpacing {
 
   spacing.gridSize = gridSize;
 
+  // Function to get the numeric value of a spacing token
+  spacing.num = (key: SpacingTokenValues): number => {
+    const pxVal = spacing[`x${key}`];
+    return parseInt(pxVal, 10);
+  };
+
+  // Design system spacing tokens
+  spacing.x0 = '0px';
+  spacing.x25 = '2px';
+  spacing.x50 = '4px';
+  spacing.x100 = '8px';
+  spacing.x150 = '12px';
+  spacing.x200 = '16px';
+  spacing.x250 = '20px';
+  spacing.x300 = '24px';
+  spacing.x400 = '32px';
+  spacing.x500 = '40px';
+  spacing.x600 = '48px';
+  spacing.x800 = '64px';
+  spacing.x1000 = '80px';
```

Comment:
we also have the problem of adding new tokens leading to plugin compatibility issues (when used from external plugins). Say from a data source plugin that needs to work on many older versions of Grafana, they would have to know to stay away from these.

Reply:
not saying we can't introduce new tokens (we can and we should when we need). Just not 100% sure introducing these are warranted, just hurts readability and ease of use. And obscures the true grid unit value. `0.5` is half a grid unit is a lot more clear than `x50' :)